src/main/haskell/kore/test/Test/Kore/OnePath/Verification.hs

```diff
@@ -262,6 +265,42 @@ test_onePathVerification = give symbolOrAliasSorts
         assertEqualWithExplanation ""
             (Left $ ExpandedPattern.fromPurePattern Mock.b)
             actual
+    , testCase "trusted second proves first" $ do
```

Comment:
Could you also add a test that trusted claims are not used in the first step? E.g. from
trusted claim: a=>b
claim: a=>b
you get a failure?